python/src/sdf/pyWorld.cc

```diff
@@ -0,0 +1,173 @@
+    .def("gravity", &sdf::World::Gravity,
+         "Get the acceleration due to gravity. The default value is "
+         "Earth's standard gravity at sea level, which equals "
+         "[0, 0, -9.80665] \f$(\frac{m}{s^2})\f$")
+    .def("set_gravity", &sdf::World::SetGravity,
+         "Set the acceleration due to gravity. Units are meters per "
+         "second squared \f$(\frac{m}{s^2})\f$")
+    .def("magnetic_field", &sdf::World::MagneticField,
+         "Get the magnetic vector in Tesla, expressed in "
+         "a coordinate frame defined by the SphericalCoordinates property. "
+         "A spherical coordinate can be specified in SDF using the "
+         "<spherical_coordinates> element.")
+    .def("set_magnetic_field", &sdf::World::SetMagneticField,
+         "Set the magnetic vector in Tesla, expressed in "
+         "a coordinate frame defined by the SphericalCoordinate. "
+         "A spherical coordinate can be specified in SDF using the "
+         "<spherical_coordinates> element.")
+    .def("spherical_coordinates", &sdf::World::SphericalCoordinates,
+         pybind11::return_value_policy::reference,
+         "Get the spherical coordinates for the world origin.")
+    .def("set_spherical_coordinates", &sdf::World::SetSphericalCoordinates,
+         "Set the spherical coordinates for the world origin.")
+    .def("model_count", &sdf::World::ModelCount,
+         "Get the number of explicit model that are immediate (not nested) "
+         "children of this World object.")
+    .def("model_by_index",
+         pybind11::overload_cast<uint64_t>(
+           &sdf::World::ModelByIndex, pybind11::const_),
+         pybind11::return_value_policy::reference,
+         "Get an immediate (not nested) child joint model on an index.")
+     .def("model_by_name",
+          pybind11::overload_cast<const std::string &>(
+            &sdf::World::ModelByName, pybind11::const_),
+          pybind11::return_value_policy::reference,
+          "Get an immediate (not nested) mutable child model based on an "
+          "index.")
+     .def("model_name_exists", &sdf::World::ModelNameExists,
+          "Get whether a model name exists.")
+     .def("add_model", &sdf::World::AddModel,
+          "Add a model to the world.")
+     // .def("add_actor", &sdf::World::AddActor,
+     //      "Add a actor to the world.")
+     // .def("add_light", &sdf::World::AddLight,
+     //      "Add a light to the world.")
+     // .def("add_physics", &sdf::World::AddPhysics,
+     //      "Add a physics object to the world.")
+     .def("add_frame", &sdf::World::AddFrame,
+          "Add a frame object to the world.")
+     .def("clear_models", &sdf::World::ClearModels,
+          "Remove all models.")
+     .def("clear_actors", &sdf::World::ClearActors,
+          "Remove all models.")
+     .def("clear_lights", &sdf::World::ClearLights,
+          "Remove all models.")
+     .def("clear_physics", &sdf::World::ClearPhysics,
+          "Remove all models.")
```

Comment:
I think we should wait to add these until we add bindings for actors, light, 
and physics.